releasenotes/notes/fix-celery-apply-async-span-close-b7a8db188459f5b5.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    tracing(celery): Fixes an issue where `celery.apply` spans didn't close if the after_task_publish or task_postrun signals didn't get called. 
```

Comment:
nit: would add a little bit about why they weren't called (exception generated in handling)

Reply:
fixed in the latest version of the release notes